db/migrate/20151110191852_create_redirections.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRedirections < ActiveRecord::Migration
+  def change
+    create_table :redirections do |t|
+      t.timestamps null: false
+      t.references :next
+      t.string :slug, index: true
+      t.string :url, length: 255
```

Comment:
Oops, 1GB: http://tinyrobots.thoughtbot.com/post/6293521440/joe-lays-down-the-law